stdlib/public/core/Range.swift

```diff
@@ -450,9 +462,13 @@ extension Range: Decodable where Bound: Decodable {
           debugDescription: "Cannot initialize \(Range.self) with a lowerBound (\(lowerBound)) greater than upperBound (\(upperBound))"))
     }
     self.init(_uncheckedBounds: (lower: lowerBound, upper: upperBound))
+    #else
+    fatalError()
```

Comment:
Is this just a todo?

Reply:
Fixed and removed, by figuring out how to avoid the #if here altogether: I needed to add an (unavailable) ExpressibleByStringInterpolation conformance to (unavailable) String.